js/packages/common/src/contexts/meta/loadAccounts.ts

```diff
@@ -276,9 +277,10 @@ export const loadAccounts = async (connection: Connection) => {
       forEachAccount(processVaultData),
     );
   const loadAuctions = () =>
-    getProgramAccounts(connection, AUCTION_ID).then(
-      forEachAccount(processAuctions),
-    );
+    pullAuctions({
+      connection,
+      auctionManagersByAuction: state.auctionManagersByAuction,
+    }).then(forEachAccount(processAuctions));
```

Comment:
```
      // bidder metadata
      getProgramAccounts(connection, AUCTION_ID, {
        filters: [
          {
            memcmp: {
              offset: 32,
              bytes: auctionManager.info.auction,
            },
          },
        ],
      }).then(forEach(processAuctions)),
      // bidder pot
      getProgramAccounts(connection, AUCTION_ID, {
        filters: [
          {
            memcmp: {
              offset: 64,
              bytes: auctionManager.info.auction,
            },
          },
        ],
      }).then(forEach(processAuctions)),
```

I believe this is what is missing from the auction state now. Current update just getting auctions from the auction program but not bidder pots and bidder metata. I'm looking into adding those queries.